tofu/imas2tofu/_core.py

```diff
@@ -2489,7 +2598,8 @@ def _checkformat_mesh_Rect(R, Z, datashape=None,
         assert R.ndim in [1, 2]
         assert Z.ndim in [1, 2]
         shapeu = np.unique(np.r_[R.shape, Z.shape])
-        shapeRZ = [None, None]
+        if shapeRZ is None:
+            shapeRZ = [None, None]
```

Comment:
should shapeRZ not be a tuple instead of a list (see below)

Reply:
ultmately, shapeRZ will be returned as a tuple, yes,
But at this point we have to define it as a list because we may have to modify it, and tuples are immutable.
So we define it as a list, possibly modify it, and finally convert it to a tuple